lib-rpc/ocamlformat_rpc_lib.mli

```diff
@@ -28,25 +28,11 @@ module type IO = sig
   (** [oc] represents an output channel *)
   type oc
 
-  val read_line : ic -> string option t
-  (** [read_line ic] will read a single line terminated by CR or CRLF from
-      the input channel [ic]. It returns {!None} if EOF or other error
-      condition is reached. *)
-
-  val read : ic -> int -> string t
-  (** [read ic len] will block until a maximum of [len] characters are read
-      from the input channel [ic]. It returns an empty string if EOF or some
-      other error condition occurs on the input channel, and can also return
-      fewer than [len] characters if input buffering is not sufficient to
-      satisfy the request. *)
-
-  val write : oc -> string -> unit t
-  (** [write oc s] will block until the complete [s] string is written to the
-      output channel [oc]. *)
-
-  val flush : oc -> unit t
-  (** [flush oc] will return when all previously buffered content from
-      calling {!write} have been written to the output channel [oc]. *)
+  val read : ic -> Sexplib0.Sexp.t option t
```

Comment:
Could you use the types from `csexp` instead if you don't mind? E.g. `Csexp.t`.

Reply:
Done.